src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/DateTimeConverter.cs

```diff
@@ -42,7 +42,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
                 dateTime = dt;
             }
 
-            if (string.Equals(culture.Name, "ja-JP"))
+            if (string.Equals(culture.Name, "ja-JP", StringComparison.OrdinalIgnoreCase))
```

Comment:
I know this won't break anything, but I think we should prefer ordinal in places where there's no clear documentation about casing.

Reply:
I did a better investigation this time and realized that the methods `.Equals()`, `.EndsWith()`, `.StartsWith()` and `.IndexOf()` when not specifying the StringComparison is going to use `Ordinal` as default. So in order to not change current behavior I'm going to use Ordinal instead

https://learn.microsoft.com/en-us/dotnet/api/system.string.equals?view=net-6.0#system-string-equals(system-string)
https://learn.microsoft.com/en-us/dotnet/api/system.string.endswith?view=net-7.0#system-string-endswith(system-char)
https://learn.microsoft.com/en-us/dotnet/api/system.string.startswith?view=net-7.0#system-string-startswith(system-string)
https://learn.microsoft.com/en-us/dotnet/api/system.string.indexof?view=net-7.0#system-string-indexof(system-string)